meta-xt-rpi5-prod-devel-domu/recipes-kernel/linux/files/defconfig

```diff
@@ -0,0 +1,3127 @@
+#
```

Comment:
I don't think it is good idea to use full .config as defconfig. You can create correct defconfig with command `make savedefconfig`

Reply:
Well, you are right in principle. But Yocto expects full `.config` as `defconfig`. Itrued to create proper defconfig with `savedefconfig` but Yocto just fails to apply it properly.

Reply:
May be put that saved defconfig as config fragment? It is very complicated task to read that.

Reply:
Created task for it https://github.com/orgs/xen-troops/projects/6/views/1?pane=issue&itemId=66374499